client/src/pages/Login.jsx

```diff
@@ -1,23 +1,62 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-
-
-const Login = () => {
-  return (
-    <div className='auth'>
-        <h1>Login</h1>
-        <form>
-            <input required type="text" placeholder="username"/>
-            <input required type="password" placeholder="password"/>
-            <button type="submit">Login</button>
-            <p>This is an error!</p>
-            <span>
-                No account? 
-                <Link to="/register">Register</Link>
-            </span>
-        </form>
-    </div>
-  )
-}
-
+import React, { useState } from 'react';
+import axios from 'axios';
+import HomeLoginCard from '../components/HomeLoginCard';
+
+
+const Login = () => {
+
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    console.log("handle submit");
+    e.preventDefault();
+
+    try {
+      const response = await axios.post('/auth/login', {
+        username,
+        password
+      });
+
+      
+      console.log('Login successful!', response.data);

```

Comment:
remove unnecessary log

Reply:
I believe this should also be kept, as it shows the session data.